modules/localgov_campaigns_paragraphs/localgov_campaigns_paragraphs.module

```diff
@@ -76,4 +84,42 @@ function localgov_campaigns_paragraphs_preprocess_paragraph(&$variables) {
       $variables['field_video_url'] = $video['uri'];
     }
   }
+
+  if ($paragraph->bundle() === 'localgov_accordion' || $paragraph->bundle() === 'localgov_accordion_pane') {
+    if (!$paragraph->get('localgov_title')->isEmpty()) {
+      $heading_text = $paragraph->get('localgov_title')->value;
+      $heading_level = $paragraph->get('field_heading_level')->value;
+      $variables['heading'] = _create_heading($heading_text, $heading_level);
+    }
+  }
+}
+
+/**
+ * Create a heading element.
+ *
+ * @param string $heading_text
+ *   Heading text.
+ * @param string $heading_level
+ *   Heading level.
+ *
+ * @return void|array
+ *   Heading render element.
+ */
+function _create_heading(string $heading_text, string $heading_level) {
```

Comment:
Good morning Gosia :)  If you don't mind, can we have this function be called _localgov_campaigns_paragraphs_create_heading().

Reply:
Done! :)